### PR TITLE
Form.io

### PR DESCRIPTION
**Pull request by aqidd**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/formio.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          label: SMTP port
+          defaultValue: '587'
+          description: The port for your smtp server
+        - id: $$cap_smtp_user
+          label: SMTP user
+          defaultValue: your_email@gmail.com
+          description: The user for your smtp server
+        - id: $$cap_smtp_pass
+          label: SMTP password
+          defaultValue: sTr0nG_p@s$wo%d
```
Comment (githubsaturn) [inline public/v4/apps/formio.yml:83]: Use hex generator, example:
https://github.com/caprover/one-click-apps/blob/a37430b15cec4ba4bceab3d434f608536fe973b8/public/v4/apps/whoopsmonitor.yml#L143

File: public/v4/apps/formio.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          label: SMPT host
+          defaultValue: smtp.gmail.com
+          description: The host for your smtp server
+        - id: $$cap_smtp_port
+          label: SMTP port
+          defaultValue: '587'
+          description: The port for your smtp server
+        - id: $$cap_smtp_user
+          label: SMTP user
+          defaultValue: your_email@gmail.com
```
Comment (githubsaturn) [inline public/v4/apps/formio.yml:79]: Remove the default value to indicate that the user needs to modify this, maybe move it to label.

Same with other default values that are not actually default values, and they are just examples. Let's move them to "label"

**Review by githubsaturn — COMMENTED** [review]
Thanks! Just some minor comments.

File: public/v4/apps/formio.yml (hunk 2 above)
Comment (aqidd) [reply public/v4/apps/formio.yml]: Thanks! I decided to remove all the default values for the SMTP as they already have a label

**Comment by githubsaturn** [comment]
Thanks

**Pull request merged**